scripts/eurostat/health_determinants/Alcohol_Consumption/process_test.py

```diff
@@ -0,0 +1,101 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Script to automate the testing for USA Population by Race preprocess script.
+"""
+
+import os
+from os import path
+import unittest
+from process import EuroStatAlcoholConsumption
+# module_dir_ is the path to where this test is running from.
+module_dir_ = os.path.dirname(__file__)
+test_data_folder = os.path.join(module_dir_, "test_data")
+op_data_folder = os.path.join(module_dir_, "test_output_data")
+input_path = os.path.dirname(os.path.abspath(__file__)) + os.sep + "input_data"
+
+
+class TestPreprocess(unittest.TestCase):
+    """
+    TestPreprocess is inherting unittest class
+    properties which further requried for unit testing
+    """
+    cleaned_csv_file_path = os.path.join(op_data_folder, "data.csv")
+    mcf_file_path = os.path.join(op_data_folder, "test_census.mcf")
+    tmcf_file_path = os.path.join(op_data_folder, "test_census.tmcf")
+
+    ip_data = [
+        os.path.dirname(os.path.abspath(__file__)) + os.sep +
+        'test_data/hlth_ehis_al1e.tsv',
+        os.path.dirname(os.path.abspath(__file__)) + os.sep +
+        'test_data/hlth_ehis_de6.tsv'
+    ]
```

Comment:
Can you please cover all functions. Probably you might have to use sample of all files.